swarm/storage/encryption/encryption_test.go

```diff
@@ -0,0 +1,125 @@
+
+var hashFunc = sha3.NewKeccak256
+
+func TestEncryptDataLongerThanPadding(t *testing.T) {
+	enc := New(4095, uint32(0), hashFunc)
+
+	data := make([]byte, 4096)
+	key := make([]byte, 32)
+
+	_, err := enc.Encrypt(data, key)
+	if err == nil || !strings.Contains(err.Error(), "Data length longer than padding") {
```

Comment:
why use contains and not `err.Error() != expMsg`

Reply:
Fixed in 62754089c8e901664f1ba003d1dab32ed2b33aa0